### ISSUE TITLE
/* stylelint-disable */ usage prevents --fix from working on entire file in v13

### ISSUE THREAD

**Issue by ryanbarkernz** [stylelint/stylelint#5282]
Say we run fix on the below:
```
body {
/* stylelint-disable */
color: red !important;
/* stylelint-enable */
background: green



}
```

The whitespace and missing semicolon will not automatically fix, though will display as errors in the console.

Declaring the actual rule seems to still allow fixing to work. EG
```
body {
// stylelint-disable declaration-no-important
color: red !important;
background: green



}
```

Downgrading to v12 fixed the issue.

**Comment by ybiquitous** [comment]
@ryanbarkernz The behavior seems due to auto-fix limitation since 13.12.0. See also:
- https://github.com/stylelint/stylelint/issues/2643#issuecomment-586622917
- https://stylelint.io/user-guide/usage/options#fix

**Comment by jeddy3** [comment]
Downgrading to v12 will reintroduce the unsafe behaviour of autofix making changes even if stylelint-disable is being used.

Please consider [contributing](https://stylelint.io/contributing) to #2643 if you have time.
